custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```
@@ -435,6 +574,34 @@
             "label": "Train / Test Set",
             "type": "SEPARATOR"
         },
+        {
+            "name": "sampling_method",
+            "label": "Sampling method",
+            "type": "SELECT",
+            "mandatory": true,
+            "description": "",
+            "selectChoices": [
+                {
+                    "value": "last_records",
+                    "label": "Last records (most recent)"
+                },
+                {
+                    "value": "no_sampling",
+                    "label": "No sampling (whole data)"
+                }
+            ],
+            "defaultValue": "last_records"
+        },
+        {
+            "name": "number_records",
+            "label": "Nb. records",
+            "description": "Number of records to use per timeseries",
```

Personally, not a fan of making "timeseries" into a portmanteau word. Just a convention, but if we were to follow it, then our users should be called "datascientist" etc.
```suggestion
            "description": "Number of last records to extract per time series",
```

---

ah good point ! i will make it into 2 words

---

545a70f